nvi

```diff
@@ -100,24 +101,57 @@ function install_node() {
   mkdir --parents -- "$BIN_DIR"
 
   # Download and extract the tar if it doesn't exist on disk.
-  if [[ ! -d "$INSTALL_DIR/node-v$NODE_VERSION-linux-x64" ]]; then
+  if [[ ! -d "$INSTALL_DIR/$PACKAGE" ]]; then
     # Fail if version doesn't exist.
-    NODE_DIST_LOOKUP=$(wget -qO - https://nodejs.org/dist/index.json | grep --null-data --only-matching --perl-regex --text "\"version\"\s*:\s*\"v$NODE_VERSION\"")
-    [[ -z "$NODE_DIST_LOOKUP" ]] && >&2 echo "Error: Couldn't find Node.js version v$NODE_VERSION. Aborting." && exit 1
+    local NODE_DIST_LOOKUP
```

Comment:
I think that's what I'll aim for. It works fine here because you can't well change neither the pattern nor `--perl-regex` without changing the other, but paired arguments have a bad habit of changing independently.